src/style.css

```
@@ -1,3 +1,72 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo__link {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  display: flex;
+  flex-direction: row;
+  margin: 0;
+  padding: 0;
+}
+
+li{
+  list-style-type: none;
+}
+
+:root {
+  --color: #00ACDC;
+}
+
+.nav__item {
+  display: block;
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0
+}
+
+.nav__link {
+  display: block;
+  text-decoration: none;
+  color: black;
+  margin: 0;
+  text-align: center;
+  line-height: 60px;
```

```suggestion
  line-height: 60px;
  font-size: 12px
```
setting right font-size from design can help to fix tests.